src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -213,30 +213,34 @@
           </div>
         </fieldset>
 
-        <!-- Locking -->
+        <!-- Object Locking -->
         <fieldset>
           <legend class="cd-header"
-                  i18n>Locking</legend>
-
-          <!-- Locking enabled -->
+                  i18n>
+            Object Locking
+            <cd-helper>
+              Store objects using a write-once-read-many (WORM) model to help you prevent objects from being deleted or overwritten for a fixed amount of time or indefinitely.
+              <br/>Object Locking works only in versioned buckets.
+            </cd-helper>
```

Comment:
what about putting this as a muted text under the heading? Its just a wild suggestion (just me thinking if we could replace all the cd-helper with the muted-text.

Reply:
![image](https://github.com/ceph/ceph/assets/25664409/24f58196-08d6-4ea5-998d-3ddeb5bcaa1f)

Reply:
this looks good to me.